test/AxelarGateway.js

```diff
@@ -728,6 +730,42 @@ describe('AxelarGateway', () => {
                     expect(balance).to.eq(0);
                 });
         });
+
+        it('should have the same deposit handler bytecode', async () => {
+            const expected = {
+                istanbul: '0x352c0ce048c2b25b0b6a58f4695613b587f3086b63b4c3a24d22c043aed230d2',
+                berlin: '0xa26b1094ee475518c006cba8bd976fd4d3cd9a6089bcbe4453b1b4cf7f095609',
+                london: '0x9f217a79e864028081339cfcead3c3d1fe92e237fcbe9468d6bb4d1da7aa6352',
+            }[EVM_VERSION]
+
+            await expect(keccak256(depositHandlerFactory.bytecode)).to.be.equal(
+              expected,
+            );
+        });
+
+        it('should have the same token bytecode', async () => {
+            const tokenFactory = await ethers.getContractFactory('BurnableMintableCappedERC20', owner);
+
+            const expectedToken = {
+                istanbul: '0xfc2522491a56af4f3519968ed49c9ba82abc79798afe8f763f601e7d5e14bdbf',
+                berlin: '0x81f6049561587bf700c0af132c504b22d696a6acfa606eee0257f92fd4ebd865',
+                london: '0x37be59a866fd46ec4179e243e5d5e2639ca1e842b152e45a34628dad6494b94b',
+            }[EVM_VERSION]
+
+            await expect(keccak256(tokenFactory.bytecode)).to.be.equal(
+              expectedToken,
+            );
+
+            const expectedDeployer = {
+                istanbul: '0xc68014e297eb42dbde383254ef3129d59528159e6c51b4f9a38f995be1dd451f',
+                berlin: '0xd3a39792ca8d1ce8e5318135ca29d8a7f0b800837726997b132ebc04f88cf9aa',
+                london: '0x0698929742de660596af20d09d04eb91bfe532ef5e2927858e4c4952034967a5',
+            }[EVM_VERSION]
+
+            await expect(keccak256(tokenDeployerFactory.bytecode)).to.be.equal(
+              expectedDeployer,
+            );
+        });
     });
```

Comment:
Can you add the AxelarGatewayProxy and gateway implementation bytecode too? While we don't need it for deriving the same address, we do want to be alerted about any unintentional changes